base/abstractarray.jl

```diff
@@ -60,7 +60,7 @@ end
 # Performance optimization: get rid of a branch on `d` in `indices(A,
 # d)` for d=1. 1d arrays are heavily used, and the first dimension
 # comes up in other applications.
-indices1{T}(A::AbstractArray{T,0}) = OneTo(1)
+indices1(A::AbstractArray{T,0}) where {T} = OneTo(1)
```

Comment:
Given this is not using `T`, this could be `indices1(A::AbstractArray{<:Any,0}) = OneTo(1)`